.envs/.production/.stupoli

```diff
@@ -0,0 +1 @@
+DJANGO_SETTINGS_MODULE=config.settings.production-stupoli
```

Comment:
There is generally a convention in python filenames not to use a hyphen, because you `-` is interpreted as a minus and as such the token `production-stupoli` is interpreted as `production` minus `stupoli`. Should probably be `production_stupoli`. Or, honestly, I'm not sure we need a separate stupoli local environment. We should maybe discuss how deployments are laid out...

I notice that I (or someone else?) did this for `production-umkc`, though, and so I can see why you did it that way...

Reply:
It was my idea to create production-umkc and local-umkc. I actually use local-umkc.yml a ton in order to test the UMKC settings fo hiding certain elements of the UI locally. I agree that it should be underscored